workflows/upstream/upstream.wdl

```diff
@@ -132,6 +123,19 @@ workflow upstream {
       default_runtime_attributes   = default_runtime_attributes
   }
 
+  call Sawfish.sawfish_discover {
+    input:
+      sex                 = select_first([sex, mosdepth.inferred_sex]),
+      aligned_bam         = aligned_bam_data,
+      aligned_bam_index   = aligned_bam_index,
+      ref_fasta           = ref_map["fasta"],                           # !FileCoercion
+      ref_index           = ref_map["fasta_index"],                     # !FileCoercion
+      out_prefix          = "~{sample_id}.~{ref_map['name']}",
+      expected_male_bed   = ref_map["hificnv_expected_bed_male"],       # !FileCoercion
+      expected_female_bed = ref_map["hificnv_expected_bed_female"],     # !FileCoercion
```

Comment:
I still a bit on the fence about using the XX and XY ploidy tracks for sawfish in deployment (at least as implemented today). The "0" and "1" GT values on the sex chromosomes seem to be unexpected by a number of downstream pipelines, so this introduces some downstream risk with a pretty minor benefit.

Reply:
How does HiPhase handle haploid genotypes? @holtjma

Reply:
It will pass them through, we already do this for TRGT

Reply:
I was not thinking so much of pb pipeline tools, moreso things like truvari and various sv script bricabrac in use downstream. It's fine for now, but once some kind of CNV integration comes into play then this ploidy file will be essential, so I might make the switch to keep genotypes as 'diploid-formatted-haploid' (ie. `0/0` and `1/1`) at that point.

Reply:
For now, do you suggest dropping the ploidy support?

Reply:
Yes, I think doing so would create fewer downstream problems.